packages/ava/src/data/statistics/index.ts

```diff
@@ -0,0 +1,269 @@
+/**
```

Comment:
```suggestion
/*
```

this is a multi-line comment for the file...not jsdoc

Reply:
done